qiskit_nature/circuit/library/ansatzes/utils/fermionic_excitation_generator.py

```diff
@@ -128,6 +124,56 @@ def generate_fermionic_excitations(
         The list of excitations encoded as tuples of tuples. Each tuple in the list is a pair. The
         first tuple contains the occupied spin orbital indices whereas the second one contains the
         indices of the unoccupied spin orbitals.
+
+    Examples:
+        Generate excitations with basic inputs.
+
+        .. jupyter-execute::
```

Comment:
Did you make sure that the execution works as expected and adds the output to the documentation?

Reply:
yeah the execution works fine, but it doesn't print any output by default so wasn't sure if I should add anything to the example. Should I explicitly call `print(excitations)` in the example and put the output in the doc?

Reply:
Yes I think printing them would make sense. I guess you already assumed that given the discussion below :+1: